code/game/objects/items/weapons/defib.dm

```diff
@@ -383,7 +383,7 @@
 							defib.deductcharge(revivecost)
 							add_logs(M, user, "revived", object="defibrillator")
 						else
-							if(tplus > tlimit)
+							if(tplus > tlimit|| !H.get_int_organ(/obj/item/organ/internal/heart))
```

Comment:
Later, perhaps we can have organs take damage while in a dead body, and have that be the criterion for "too late for defib".